src/EnergyPlus/DataSystemVariables.cc

```diff
@@ -202,7 +202,9 @@ namespace DataSystemVariables {
     void CheckForActualFileName(IOFiles &ioFiles,
                                 std::string const &originalInputFileName, // name as input for object
                                 bool &FileFound,                          // Set to true if file found and is in CheckedFileName
-                                std::string &CheckedFileName              // Blank if not found.
+                                std::string &CheckedFileName,             // Blank if not found.
+                                std::vector<std::string> *pathsChecked,
```

Comment:
Please pass in a reference here instead of a pointer.

Reply:
@Myoldmopar Neal and I actually decided to eliminate this parameter. Instead we are just putting the vector in the method. This makes things a lot simpler, and keeps the usage of the function more similar.